.babelrc

```diff
@@ -14,6 +14,8 @@
         "react-hot-loader/babel",
         ["@babel/plugin-proposal-decorators", { "legacy": true }],
         ["@babel/proposal-class-properties", { "loose": true }],
+        ["@babel/plugin-proposal-private-methods", { "loose": true }],
```

Comment:
both plugins were already using "loose" setting. Put it in here allows us to avoid warnings in console. We can re-evaluate the state later.